lib/lambda_ethereum_consensus/p2p/gossip/operations_collector.ex

```diff
@@ -82,40 +87,17 @@ defmodule LambdaEthereumConsensus.P2P.Gossip.OperationsCollector do
       attestations: block.body.attestations
     }
 
-    GenServer.cast(__MODULE__, {:new_block, block.slot, operations})
+    filter_messages(block.slot, operations)
   end
 
-  @impl true
-  def handle_gossip_message(topic, msg_id, message) do
-    GenServer.cast(__MODULE__, {:gossipsub, {topic, msg_id, message}})
-  end
-
-  @impl GenServer
-  def init(_init_arg) do
+  def init() do
```

Comment:
Let's add a comment explaining that this both joins the topics and sets the operations as empty in the DB. Remember, side effects must be clear for the caller in the comments.